src/lib/styles/app.css

```
@@ -152,6 +152,12 @@ button.message-button:hover {
     --dy-tab-p: 1rem;
 }
 
+/* This corrects the center alignment of the footnotes */
+@utility dy-stack {
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@utility`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)